### ISSUE TITLE
`theme:pot` command and `load_theme_textdomain()` function are not in agreement on languages location

### ISSUE THREAD

**Issue by richaber** [WebDevStudios/wd_s#595]
The `_s_setup()` [function](https://github.com/WebDevStudios/wd_s/blob/29de439c760c4736ec603fee342f1a2293b69ec1/functions.php#L29) loads the theme's text domain from `_s/languages/`, which is a common/expected location for a theme's POT/PO/MO file(s).

```
load_theme_textdomain( '_s', get_template_directory() . '/languages' );
```

The NPM `theme:pot` [command](https://github.com/WebDevStudios/wd_s/blob/29de439c760c4736ec603fee342f1a2293b69ec1/package.json#L34) builds the POT file to `build/languages/_s.pot`, which is an _unexpected_ location.

```
"theme:pot": "wp i18n make-pot . build/languages/_s.pot --exclude=node_modules,vendor,build",
```

**Comment by coreymcollins** [comment]
@gregrickaby – do we have a preference for where these files live? Do you prefer we keep them in the `/build/languages/` directory and update `load_theme_textdomain` or move them to a `/languages/` top level directory?

**Comment by coreymcollins** [comment]
Closed by #596
